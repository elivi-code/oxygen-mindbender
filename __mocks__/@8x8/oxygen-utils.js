export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
