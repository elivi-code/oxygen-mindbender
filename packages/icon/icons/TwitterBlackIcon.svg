<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<g opacity="0.25">
<path d="M12 24C18.6274 24 24 18.6274 24 12C24 5.37258 18.6274 0 12 0C5.37258 0 0 5.37258 0 12C0 18.6274 5.37258 24 12 24Z" fill="#292F33"/>
<path d="M9.80384 18.33C15.1258 18.33 18.0358 13.92 18.0358 10.098C18.0358 9.97203 18.0358 9.84603 18.0298 9.72603C18.5938 9.31803 19.0858 8.80803 19.4758 8.22603C18.9598 8.45403 18.4018 8.61003 17.8138 8.68203C18.4138 8.32203 18.8698 7.75803 19.0858 7.08003C18.5278 7.41003 17.9098 7.65003 17.2498 7.78203C16.7218 7.21803 15.9718 6.87003 15.1378 6.87003C13.5418 6.87003 12.2458 8.16603 12.2458 9.76203C12.2458 9.99003 12.2698 10.212 12.3238 10.422C9.91784 10.302 7.78784 9.15003 6.35984 7.39803C6.11384 7.82403 5.96984 8.32203 5.96984 8.85003C5.96984 9.85203 6.47984 10.74 7.25984 11.256C6.78584 11.244 6.34184 11.112 5.95184 10.896C5.95184 10.908 5.95184 10.92 5.95184 10.932C5.95184 12.336 6.94784 13.5 8.27384 13.77C8.03384 13.836 7.77584 13.872 7.51184 13.872C7.32584 13.872 7.14584 13.854 6.96584 13.818C7.33184 14.97 8.39984 15.804 9.66584 15.828C8.67584 16.602 7.42784 17.064 6.07184 17.064C5.83784 17.064 5.60984 17.052 5.38184 17.022C6.64784 17.85 8.17184 18.33 9.80384 18.33Z" fill="white"/>
</g>
</svg>
