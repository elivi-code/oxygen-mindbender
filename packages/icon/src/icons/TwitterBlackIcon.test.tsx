/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import TwitterBlackIcon from './TwitterBlackIcon';

describe('<TwitterBlackIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<TwitterBlackIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
