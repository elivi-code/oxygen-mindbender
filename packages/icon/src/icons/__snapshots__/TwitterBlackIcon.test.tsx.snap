// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TwitterBlackIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <g
      opacity=".25"
    >
      <path
        d="M12 24c6.6274 0 12-5.3726 12-12S18.6274 0 12 0 0 5.3726 0 12s5.3726 12 12 12Z"
        fill="#292F33"
      />
      <path
        d="M9.8038 18.33c5.322 0 8.232-4.41 8.232-8.232 0-.126 0-.252-.006-.372a5.9274 5.9274 0 0 0 1.446-1.5c-.516.228-1.074.384-1.662.456.6-.36 1.056-.924 1.272-1.602-.558.33-1.176.57-1.836.702-.528-.564-1.278-.912-2.112-.912-1.596 0-2.892 1.296-2.892 2.892 0 .228.024.45.078.66-2.406-.12-4.536-1.272-5.964-3.024-.246.426-.39.924-.39 1.452 0 1.002.51 1.89 1.29 2.406-.474-.012-.918-.144-1.308-.36v.036c0 1.404.996 2.568 2.322 2.838-.24.066-.498.102-.762.102-.186 0-.366-.018-.546-.054.366 1.152 1.434 1.986 2.7 2.01-.99.774-2.238 1.236-3.594 1.236-.234 0-.462-.012-.69-.042 1.266.828 2.79 1.308 4.422 1.308Z"
        fill="#fff"
      />
    </g>
  </g>
</svg>
`;
